### DISCUSSION TITLE
Redesign the header for mobile

### DISCUSSION THREAD

**Discussion by miaw-miaw** [forem/forem#19742] [Feature Requests]
Hello,
I think we shoukd use different logi size for mobile version. It would be better while in mobile version the logo would tranform into an icon size to get more space for a “create post” button.

Right now the create poat button is hidden in user menu.

**Comment by Sarah-okolo** [comment]
Hi @miaw-miaw, I believe the  mobile header is fine just the way it is, and the "create post" button is not present directly on the header like it is on desktop view, but rather in the dropdown menu you get when you click on your profile icon in the header, making it the perfect fit for mobile view, with the search bar also transformed to just an icon, to create more space in the header. so I believe this feature you're requesting is already present.
